htdocs/lib2/translationHandler.class.php

```diff
@@ -238,7 +238,7 @@ public function substitueVariables(&$variables, $lang, $str)
 
         // replace variables in string
         if (mb_ereg_search_init($langstr)) {
-            while (false != $vars = mb_ereg_search_regs("%[^%]*%")) {
+            while ($vars = mb_ereg_search_regs('%[^%]*%') != false) {
```

Comment:
reverted changes and disabled the yoda style sniffer, because it's too dangerous and may introduce more bugs like this